dockers/docker-gnmi-watchdog/Dockerfile.j2

```diff
@@ -0,0 +1,39 @@
+FROM docker-config-engine-bookworm-{{DOCKER_USERNAME}}:{{DOCKER_USERTAG}} AS builder
+
+# Update apt's cache of available packages
+RUN apt-get update && apt-get install -y \
+    build-essential
```

Comment:
>essential [](http://example.com/codeflow?start=10&length=9)

how large is this docker image? If large, can we remove some build dependencies in the end of Dockerfile?

Reply:
I have updated this dockerfile to remove build-essential in the end